qa/tasks/ldap_client.py

```diff
@@ -0,0 +1,105 @@
+"""
+ldap_client
+"""
+import logging
+import contextlib
+import time
+import os
+
+from teuthology.orchestra import run
+from teuthology import misc
+
+log = logging.getLogger(__name__)
+
+def get_node_name(ctx, this_task):
+    """
+    Given a teuthology task site ('client.1' for example), return
+    the network hostname for that site.
+    """
+    for task_list in ctx.cluster.remotes:
+        if this_task in ctx.cluster.remotes[task_list]:
+            return task_list.name.split('@')[1]
+    return ''
+
+def get_task_site(ctx, taskname):
+    """
+    Given a taskname ('ldap_client' for example), return the associated
+    teuthology location that that task is running on ('client.1' for example)
+    """
+    for tentry in ctx.config['tasks']:
+        if taskname in tentry:
+            return tentry[taskname][0]
+    return ''
+
+def fix_yum_boto(client):
+    """
+    Set up boto on Rhel/Fedora/Centos.
+    """
```

Comment:
for safety we can check pkg type is rpm, if it can work with debian, we can add those packages too or run this test outisde multifs dir and fix the os-type.

Reply:
The rpm check is done inside the task which calls this.  I guess that it makes sense to move these lines inside the task itself.